pkg/kubectl/cmd/set/set_image.go

```diff
@@ -232,18 +250,18 @@ func (o *ImageOptions) Run() error {
 			continue
 		}
 
-		// no changes
```

Comment:
Don't change the order here. Open a separate pr addressing this for all set commands, like it was already mentioned in #62139

Reply:
Ah, completely missed reverting this change on this file. Will sweep through all other files and see if I missed reverting this elsewhere. Thanks